.travis.yml

```diff
@@ -13,8 +12,6 @@ env:
     - ACTION=loadtest
 before_script:
     - psql -c "CREATE DATABASE testdb ENCODING 'UTF8' TEMPLATE template0;" -U postgres
-before_install:
-    - sudo apt-get install postgresql-contrib
```

Comment:
wow, isn't that needed? Weird, can you confirm @Natim @leplatrem @ametaireau @n1k0